dashboard/app/models/user.rb

```diff
@@ -524,7 +524,7 @@ def sections
   validates :name, length: {within: 1..70}, allow_blank: true
   validates :name, no_utf8mb4: true
 
-  defer_age = proc {|user| %w(google_oauth2 clever powerschool).include?(user.provider) || user.sponsored?}
+  defer_age = proc {|user| %w(google_oauth2 clever powerschool).include?(user.provider) || user.sponsored? || Policies::Lti.lti?(user)}
```

Comment:
When defering age, will we still prompt a student to select their age upon first time login?

Reply:
Yeah, this only gets around the validation when the record gets saved.